recipes/pyfolio/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pyfolio" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "e30ee8c1932b45c4ce066cc78aee957d939e02a9427e797e2f7cd582cb0938c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
We are moving to `pip` for the install. Please see [the docs for details]( https://conda-forge.org/docs/recipe.html#optional-bld-bat-and-or-build-sh ). Would update this according to the recommendation there.

Reply:
I'll have a look right now and update accordingly :)